WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -118,6 +148,24 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     func didTapEditAd() {
         onEditAd?()
     }
+
+    func loadAISuggestions() async {
```

Comment:
nit: wdyt about grouping this it to be under
// MARK: - Blaze AI Suggestions

?

Reply:
Done in 044435a2a6ba501214497772c71d6b9d2dd87c1d